en/concept/mixing.md

```diff
@@ -161,6 +163,19 @@ A mixer file must be named **XXXX._main_.mix** if it is responsible for the mixi
 > **Note** `XXXX` in the mixer file name is set in the airframe definition (e.g. see `set MIXER quad_w` in [airframes/10015_tbs_discovery](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/airframes/10015_tbs_discovery))
 
 
+### Loading a Custom Mixer {#loading_custom_mixer}
+
+PX4 loads appropriately named mixer files from the SD card directory **etc/mixers**, by preference, and then the version in Firmware.
+
+To load a custom mixer, you should give it the same name as a "normal" mixer file (that is going to be loaded by your airframe) and put it in the **etc/mixers** directory on your flight controller's SD card.
+
+Most commonly you will use the mixer file for your current airframe (**XXXX._aux_.mix**), where the `XXXX` prefix is the set in the airframe configuration file (e.g. [airframes/10015_tbs_discovery](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/airframes/10015_tbs_discovery) loads **quad_w._aux_.mix** - see `set MIXER quad_w`).
```

Comment:
Might be worth noting that you can override the main and/or aux mixer

Reply:
I've clarified that you'll most commonly use the airframe *AUX* file, and added the comment about main in first note below. The point being that you can overload anything you like, but AUX in your airframe is a a good target.